packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -204,29 +242,20 @@ export class EthereumAnchorService implements AnchorService {
    * @param streamId - Stream ID
    * @param tip - Tip CID of the stream
    */
-  pollForAnchorResponse(streamId: StreamID, tip: CID): Observable<CASResponse> {
-    if (process.env.CERAMIC_DISABLE_ANCHOR_POLLING_RETRIES == 'true') {
-      return this._pollForAnchorResponseLegacy(streamId, tip)
-    } else {
-      return this._pollForAnchorResponse(streamId, tip)
-    }
+  pollForAnchorResponse(streamId: StreamID, tip: CID): Observable<AnchorEvent> {
+    return this._pollForAnchorResponse(streamId, tip)
   }
 
-  private _pollForAnchorResponse(streamId: StreamID, tip: CID): Observable<CASResponse> {
+  private _pollForAnchorResponse(streamId: StreamID, tip: CID): Observable<AnchorEvent> {
```

Comment:
nitpick: no need for this helper anymore, can fold it up into `pollforAnchorResponse`

Reply:
yup